lang/cs/Org.Apache.REEF.Network/Group/Driver/IGroupCommDriver.cs

```diff
@@ -92,5 +92,13 @@ public interface IGroupCommDriver
         /// <returns>The Group Communication task configuration with communication group and
         /// operator configuration set.</returns>
         IConfiguration GetGroupCommTaskConfiguration(string taskId);
+
+        /// <summary>
+        /// Get the service configuration required for running Group Communication on Reef tasks.
+        /// </summary>
+        /// <param name="config">User specified configuration to be passed to service</param>
+        /// <returns>The service configuration for the Reef tasks</returns>
+        IConfiguration GetServiceConfiguration(params IConfiguration[] config);
```

Comment:
We talked no need to add this API. You can always GetServiceConfiguration(), then merge it with your configuration in the client's driver.